data/examples/declaration/value/function/operator-chains-out.hs

```diff
@@ -0,0 +1,245 @@
+module A (a) where
+
+-- Template
+-- x :: Int
+-- x =
+--   1 op_a
+--   2 op_a
+--   3 op_a
+--   4 op_b
+--   5 op_a
+--   6 op_a
+--   7 op_a
+--   8
+
+-- Right chain, normal case, 1 operator type
+a :: Int
+a =
+  1
+    ++ 2
+    ++ 3
+    ++ 4
+
+-- Right chain, : case, 1 operator type
+b :: Int
+b =
+  1
```

Comment:
Is it really necessary to repeat the same operator so many times? I think just 2 operators are enough in cases like this.

Reply:
When I was working on #528 , I had inconsistencies which could only be detected in chains with 4 operators, that's why I chose 4 identical operators, or 3-1-3